twilio/task_router/taskrouter_config.py

```diff
@@ -10,8 +10,26 @@ class TaskRouterConfig:
     """
 
     def __init__(self, rules, default_target):
+
         self.filters = rules
         self.default_filter = default_target
+        self.workflow_rules = []
```

Comment:
storing filters in two variables (`filters`, `workflow_rules`) is asking for trouble. nothing keeps them consistent. Instead, we should only use `filters` at the cost of keeping the user experience as bad as it currently is.